src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -120,6 +120,15 @@ export class ExamParticipationSummaryComponent implements OnInit {
     }
 
     private tryLoadPlagiarismCaseInfosForStudent() {
+        // If the exam has not yet ended, or we're only a few minutes after the end, we can assume that there are no plagiarism cases yet.
+        // We should avoid trying to load them to reduce server load.
+        if (this.studentExam?.exam?.endDate) {
+            const endDateWithTimeExtension = dayjs(this.studentExam.exam.endDate).add(15, 'minutes');
+            if (dayjs().isBefore(endDateWithTimeExtension)) {
```

Comment:
According to the comment `!dayjs().isAfter(endDateWithTimeExtension)` would be more logical to me.